### DISCUSSION TITLE
Do emails support i18n?

### DISCUSSION THREAD

**Discussion by LeOndaz** [saleor/saleor#8996] [Q&A]
Can we translate emails?

**Comment by karolkielecki** [comment]
Yes, you should be able to translate emails.

**Comment by LeOndaz** [comment]
@karolkielecki how can that be done? assuming that you don't mean changing the template from the dashboard

**Reply by JustCreate3D** [reply]
Hi @LeOndaz, 
We were currently solving this issue too.
Basically, we found out that you have 2 options, but in either case, you'll have to edit the template from the dashboard:
1. Where 1 Channel = 1 Language, this is easy, just go to the User Email plugin configuration and rewrite into your local language the emails from the default using handlebars. Use [this](https://github.com/saleor/saleor-email-templates) repo for autogenerating templates with already saleor-set templates and just translate the fix strings. 
2. Our case where we use the "europe-eur" channel for serving all customers with € but in different languages. You create a kind of switch case with the `compare` function where you gather either the `language_code` or another attribute for identifying the language of the user like `billing_address.country` attribute from the event payload. You will also need to use the [same repo](https://github.com/saleor/saleor-email-templates) as in the 1st case for generating the whole template.

Example code:
```
{{#if (compare order.billing_address.country "==" "SK")}}
Ďakujeme za objednávku
{{else}}
Hello thank you for your order
{{/if}}
```
I found somewhere, that saleor don't plan to have an option for a more user friendly way of translating this text.  

Related docs here [https://docs.saleor.io/docs/3.0/developer/available-plugins/user-emails](https://docs.saleor.io/docs/3.0/developer/available-plugins/user-emails)

**Comment by LeOndaz** [comment]
@JustCreate3D thank you for your kind and detailed response, I'll get back to you if I got anything new
